modules/accredited_representative_portal/app/models/accredited_representative_portal/power_of_attorney_request.rb

```diff
@@ -104,11 +104,26 @@ def mark_replaced!(superseding_power_of_attorney_request)
     scope :resolved, -> { joins(:resolution) }
 
     scope :decisioned, lambda {
-      where(
-        resolution: {
-          resolving_type: PowerOfAttorneyRequestDecision.to_s
-        }
-      )
+      left_outer_joins(:resolution)
+        .where(
+          resolution: {
+            resolving_type: PowerOfAttorneyRequestDecision.to_s
+          }
+        )
+    }
+
+    scope :sorted_by, lambda { |sort_column, direction = :asc|
+      case sort_column
+      when 'created_at'
+        order(created_at: direction)
+      when 'resolved_at'
+        safe_direction = direction.to_sym == :asc ? 'ASC' : 'DESC'
+        includes(:resolution)
+          .references(:resolution)
+          .order(Arel.sql("ar_power_of_attorney_request_resolutions.created_at #{safe_direction} NULLS LAST"))
```

Comment:
Rather than vanilla string interpolation here, could we either hard code the SQL snippets for each branch or make use of one of the `sanitize_*` methods from `ActiveRecord`. Just a safety concern here for what statements are issued to the DB.

Reply:
should the treatment of nulls in the ordering be related to the ordering being asked for (asc vs desc)?

Reply:
ok I'm doing this:

`      null_treatment = normalized_order == 'asc' ? 'NULLS LAST' : 'NULLS FIRST'
`